eng/Export-API.ps1

```diff
@@ -1,3 +1,10 @@
+[CmdletBinding()]
+param (
+    [Parameter(Position=0)]
```

Comment:
Does this definition allow calling the script without parameters?

Reply:
Yep, and I tested it as well just to make sure everything flows correctly. This also makes it support, out of box, `-Verbose` and friends, in case you ever wanted to use `Write-Verbose`, etc., as well.

Reply:
Don't you need to set a default value if you don't set it? Or does that happen automatically because it is a positional parameter?

Reply:
No, because it's not mandatory. `ValidateNotNullOrEmpty` only applies to values passed to it explicitly. Plus I check it further down in a conditional (PowerShell uses lazy conditionals like JS - the object has a value; otherwise, null, empty, and 0 are all evaluated to `$false`).